src/Controls/src/Core/HandlerImpl/Element.Impl.cs

```diff
@@ -1,13 +1,17 @@
 ﻿using System;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Maui.Controls.Hosting;
 
 namespace Microsoft.Maui.Controls
 {
-	public partial class Element : Maui.IElement
+	public partial class Element : Maui.IElement, IEffectControlProvider
```

Comment:
In XF all the renderers implemented `IEffectControlProvider` so this just consolidates that behavior to the Element instead of the renderer